layouts/shortcodes/live-telemetry.html

```diff
@@ -1,60 +1,59 @@
 <aside class="telemetry" id="live">
-  <h2>Livedaten aus der Community</h2>
+  <h2>{{ i18n "livedaten_aus_der_community" }}</h2>
   <div class="telemetry_inner">
     <div class="telemetry_box">
       <shopicon-regular-car3 class="telemetry_icon" size="xl"></shopicon-regular-car3>
       <div class="telemetry_details">
         <div class="telemetry_main">
           <div class="telemetry_title">
-            Ladeleistung
+            {{ i18n "ladeleistung" }}
           </div>
           <div class="telemetry_value">
             <span id="telemetry--power"></span>
             <span class="telemetry_unit">kW</span>
           </div>
 
         </div>
-        <div class="telemetry_sub"><span id="telemetry--activeClients"></span> aktive Nutzer</div>
+        <div class="telemetry_sub"><span id="telemetry--activeClients"></span> {{ i18n "aktive_nutzer" }}</div>
```

Comment:
🤮 Feels like the better way to do this is to have a translated version of this entire shortcode, but I'm not sure what the best way to go about that is.

Reply:
Similar issue at line 54

Reply:
Resolved in 81449ca